geoProcess.py

```diff
@@ -0,0 +1,275 @@
+import json
+import numpy as np
+import sys, os
+from area import area
+import argparse
+
+def update_function(old_value, new_value, flag):
```

Comment:
Let's name this something more descriptive + add a docstring to describe it's utility.

Reply:
re-named the function already.